docs/source/AdministratorGuide/Configuration/ConfReference/Systems/WorkloadManagement/Agents/PilotStatusAgent/index.rst

```diff
@@ -8,9 +8,6 @@ Special attributes for this agent are:
 +--------------------------+--------------------------------------------+-----------------------------------+
 | **Name**                 | **Description**                            | **Example**                       |
 +--------------------------+--------------------------------------------+-----------------------------------+
-| *GridEnv*                | Path where is located the file to          | GridEnv = /usr/profile.d/grid-env |
```

Comment:
You deleted `GridEnv` from the documentation of the `PilotStatusAgent` but not from the agent itself:

https://github.com/DIRACGrid/DIRAC/blob/16c620ed0446788e552067b439519c3de757c9a0/src/DIRAC/WorkloadManagementSystem/Agent/PilotStatusAgent.py#L48

Is this expected?
It also appears here: 

https://github.com/DIRACGrid/DIRAC/blob/16c620ed0446788e552067b439519c3de757c9a0/docs/source/AdministratorGuide/HowTo/multiVO.rst#workloadmanagement---pilotstatusagent

Reply:
Good catch! Solved, thanks